keras_app/views/layers_export.py

```diff
@@ -569,6 +570,15 @@ def batch_norm(layer, layer_in, layerId, idNext, nextLayer,):
                                           scale=False, center=False)(*layer_in)
     return out
 
+def bidirectional(layer, layer_in, layerId):
+    out = {layerId: Bidirectional(layer)(*layer_in)}
```

Comment:
Why are you creating the layer twice? We should first get all parameters and then create the layer once assigning all the parameter values.